dask/array/reductions.py

```diff
@@ -37,11 +37,92 @@ def divide(a, b, dtype=None):
+        If the reduction can be performed in less than 3 steps, it will not
+        be invoked at all.
+    aggregate: callable(x_chunk, axis, keepdims)
+        Last kernel function to be executed when resolving the dask graph,
+        producing the final output. It is always invoked, even when the reduced
+        Array counts a single chunk along the reduced axes.
+    axis: int or sequence of ints, optional
+        Axis or axes to aggregate upon. If omitted, aggregate along all axes.
+    keepdims: boolean, optional
+        Whether the reduction function should preserve the reduced axes,
+        leaving them at size ``output_size``, or remove them.
+    dtype: np.dtype, optional
+        Force output dtype. Defaults to x.dtype if omitted.
+    split_every: int >= 2 or dict(axis: int), optional
+        Determines the depth of the recursive aggregation. If set to or more
+        than the number of input chunks, the aggregation will be performed in
+        two steps, one ``chunk ``kernel per input chunk and a single
+        ``aggregate`` kernel at the end. If set to less than that, an
+        intermediate ``combine`` kernel will be used, so that any one
+        ``combine`` or ``aggregate`` kernel has no more than ``split_every``
+        inputs. The depth of the aggregation graph will be
+        :math:`log_{split_every}(input chunks along reduced axes)`. Setting to
+        a low value can reduce cache size and network transfers, at the cost of
+        more CPU and a larger dask graph.
+
+        Omit to let dask heuristically decide a good default. A default can
+        also be set globally with the ``split_every`` key in
+        :mod:`dask.config`.
+    name: str, optional
+        Prefix of the keys of the intermediate and output nodes. If omitted it
+        defaults to the kernel function names.
+    out: Array, optional
+        Another dask array whose contents will be replaced. Omit to create a
+        new one. Note that, unlike in numpy, this setting gives no performance
+        benefits whatsoever, but can still be useful  if one needs to preserve
+        the references to a previously existing Array.
+    concatenate: bool, optional
+        If True (the default), the output of the kernels is concatenated into a
+        single np.array before being passed to the ``combine`` and
+        ``aggregate`` kernels. If False, the input of ``combine`` and
+        ``aggregate`` will be either a list of the raw outputs of the previous
+        step or a single output, and the kernel will have to concatenate it
+        itself. It can be useful to set this to False if the chunk and/or
+        combine steps do not produce np.arrays.
+    output_size: int >= 1, optional
+        Size of the output of the ``aggregate`` kernel along the reduced axes.
+        Ignored if keepdims is False.
+
+    Returns
+    -------
+    dask array
+
+    Kernel Parameters
+    -----------------
```

Comment:
My experience has been that sphinx tends to  drop section headers that don't match one of their known set.  I tend to revert to using boldface instead like `**Kernel Parameters**`.   It would be good to check that things haven't changed though.